options.go

```diff
@@ -110,6 +112,24 @@ func (o *Options) SetUseGZip(useGZip bool) *Options {
 	return o
 }
 
+// HTTPClient returns the http.Client that is configured to be used
+// for HTTP requests. It will return the one that has been set using
+// SetHTTPClient or it will construct a default client using the
+// other configured options.
+func (o *Options) HTTPClient() *nethttp.Client {
+	return o.httpOptions.HTTPClient()
+}
```

Comment:
This should be tested in the `/options_test.go`

Reply:
Done.